src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -781,7 +781,7 @@ std::string servicePathSubscriptionRegex(const std::string servicePath, std::vec
 
   if (spathComponents == 0)
   {
-    spathRegex = "^$|^\\/#$|^\\/$";  // FIXME NOW: Including empty service path also, not sure it's correct
```

Comment:
This hasn't to do with this PR but I have spoted looking to code. I think is an old leftover, so I removed it.

NTC